packages/grid/test/sorting.test.js

```diff
@@ -351,6 +351,41 @@ describe('sorting', () => {
       });
     });
 
+    describe('multi-sort priority', () => {
```

Comment:
Would suggest to name this suite so that it indicates that it tests for the `append` mode specifically.

I assume that the `prepend` mode is covered implicitely by the existing tests?

Reply:
Done. Yes, the `prepend` value is basically same as no value at all.